types/src/on_chain_config/execution_config.rs

```diff
@@ -106,7 +106,7 @@ pub struct ExecutionConfigV3 {
 impl Default for ExecutionConfigV3 {
     fn default() -> Self {
         Self {
-            transaction_shuffler_type: TransactionShufflerType::NoShuffling,
+            transaction_shuffler_type: TransactionShufflerType::SenderAwareV2(32),
             block_gas_limit: None,
```

Comment:
@danielxiangzl - Should we enable block gas limit by default for Forge and other testings?

Reply:
Yes, but only after the execution on-chain config registry is fixed. I still find it hacky to set the config values via default for test networks. Maybe do what Igor suggested and set the block gas limit to be 35000 for genesis?

Reply:
This will be used only for Forge and existing integration tests, so I say we enable gas limit so that our tests exercise the block gas limit before rolling this out. For testnet/mainnet, this is no-op as they are still using the V1.

Reply:
Actually I checked and we are on V3 in mainnet and testnet. But it still shouldn't matter, right?

Reply:
If the execution on-chain config registry is not fixed, and main/testnet reads config via default (V3), then it will be a problem. So just to double check, is it fixed already?

Reply:
How are we on V3 in mainnet? If so, I think this is a dangerous change to make..

Reply:
The governance proposal for v1.5 was V3

Reply:
Since the registry fix is not rolled out yet, I think we should not enable shuffling by default in V3 - if we are already in V3 in mainnet. The reason for that being if by any chance we roll out this change without the registry fix, then we break the network.

Reply:
Let's add V4 below Missing, and return no shuffling in v3? Then we don't need new shuffling enum either